web/src/nav/StatusMessagesNavItem.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { setLinkComponent } from '../../../shared/src/components/Link'
+import * as GQL from '../../../shared/src/graphql/schema'
+import { StatusMessagesNavItem } from './StatusMessagesNavItem'
+
+describe('StatusMessagesNavItem', () => {
+    setLinkComponent((props: any) => <a {...props} />)
+    afterAll(() => setLinkComponent(null as any)) // reset global env for other tests
```

Comment:
```suggestion
    afterAll(() => setLinkComponent(() => null)) // reset global env for other tests
```

This behaves the same in my testing and is type safe